docs/conf.py

```diff
@@ -0,0 +1,59 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("."))
```

Comment:
Why adding "." which is `docs/`
Maybe you want to do this:
```suggestion
sys.path.insert(0, os.path.abspath(".."))

import h5grove  # noqa
```

Reply:
The only advantage of adding `".."` is that you don't need to install `h5grove` to build the doc.
Without it, when fixing the doc, you have to reinstall the package before rebuilding the doc.